realm/realm-library/src/main/java/io/realm/RealmAnyType.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm;
+
+import org.bson.types.Decimal128;
+import org.bson.types.ObjectId;
+
+import java.util.Date;
+import java.util.UUID;
+
+import javax.annotation.Nullable;
+
+import static io.realm.RealmFieldTypeConstants.MAX_CORE_TYPE_VALUE;
+
+
+public enum RealmAnyType {
```

Comment:
Since this enum only make sense in the context of RealmAny....would it make sense to move it to `RealmAny.Type` ... similar to what we have done on e.g. `SyncSession`

Reply:
Much better :)